geode-core/src/main/java/org/apache/geode/internal/cache/InternalRegion.java

```diff
@@ -459,4 +459,7 @@ VersionedObjectList basicRemoveAll(Collection<Object> keys,
    * @return true if synchronization should be attempted
    */
   boolean shouldSyncForCrashedMember(InternalDistributedMember id);
+
+  void clearRegionLocal(RegionEventImpl regionEvent, boolean cacheWrite,
```

Comment:
Maybe it's just me, but the old method name , `clearRegionLocally`, seems to be easier to understand for the reader?.